src/cli/backup/sharepoint_e2e_test.go

```diff
@@ -341,7 +342,9 @@ func runSharepointDetailsCmdTest(suite *PreparedBackupSharepointE2ESuite, catego
 	foundList := 0
 
 	for _, ent := range deets.Entries {
-		if ent.SharePoint != nil && ent.SharePoint.ItemName != "" {
+		if ent.SharePoint != nil &&
```

Comment:
Looks like we ended up shipping it for items that are not lists.

Reply:
@meain  attempted to make it generic instead of renaming the function

Reply:
Hmm, not sure if we are on the same page here. The function was previously checking for details of SharePoint drive items and hence `ent.SharePoint.ItemName` check, but now that case is never checked as `ent.SharePoint.List` will be `nil` for drive items.

Reply:
@meain please take a look at the current change. Now it caters to both sharepoint list and drive items.